### DISCUSSION TITLE
[Scam Alert] Taste NFT

### DISCUSSION THREAD

**Discussion by Webamoozcom** [Webamoozcom/warning-list#34] [Announcements]
<div dir="rtl">

**Project Name:** 
Taste NFT

**نام پروژه:** 
Taste NFT

**نماد پروژه:**
TASTE 

**اتفاقی که افتاده است:**

اقای اوسیوند که  کانال تو بی تی سی رو دارن. ( که تو پروژه بریج سینا استوی کمک خیلی بزرگی بهش کرد و همش در حال تشویق به خرید بریج بود ) 12 آگوست اومد یه شت کوینی رو معرفی کرد.

BRG
شت کوین Taste NFT 
https://t.me/TOBTC/8084

![photo_2021-09-11_08-08-25](https://user-images.githubusercontent.com/87842536/132935021-e1b8d576-05c7-4a67-bd59-932127344829.jpg)

مشخصه که ایشون خودشون کاملا پشت این پروژه هستن و برای ظاهر سازی یه چند تا خارجی اوردن جلو. مدارک رو میزارم👇🏻
![photo_2021-08-20_23-50-44](https://user-images.githubusercontent.com/87842536/132934887-e7edf76e-4d4e-46e4-8269-100f838e7889.jpg)

بین 11 می تا 15 زمانی که هنوز حتی وایت پیپر هم نداشت! 6 کا ممبر میگیره گروهش ( فیکن)

![photo_2021-08-21_00-34-03](https://user-images.githubusercontent.com/87842536/132934903-81c34ab0-edf2-4d43-b32b-8f289ecea60f.jpg)

نصف ممبر های گروهش ایرانین: 
https://t.me/TasteNFT

اینا کسایین که به عنوان تیم گذاشته داخل وبسایت ( هویتشون واقعیه ولی پول داده بهشون که اینارو در معرض دید بزاره )
![photo_2021-09-11_08-04-09](https://user-images.githubusercontent.com/87842536/132934931-fa2c7025-d1e1-40a1-ac7d-e4644b35b995.jpg)

آقای Michael Oates که تا پارسال آینه ماشین تعویض میکرد فیلم میگرفت میزاشت یوتیوب تا از تبلیغاتش پول بگیره الان اومده شده ceo & founder توکن؟!
![photo_2021-09-11_08-04-52](https://user-images.githubusercontent.com/87842536/132934954-a96e69a6-5b38-41fb-8a64-1665c36b0fe2.jpg)

ایشون هم دانشجوی مهندسی پزشکی هستن. چطور شدن cto
![photo_2021-09-11_08-05-45](https://user-images.githubusercontent.com/87842536/132934968-8d8fce90-6a86-425a-8aa1-98f1b80f261b.jpg)

توی یک ویدیویی که مایکل 6 سال پیش آپلود کرده این cto هم دیده میشه. احتمالا زنشه.
![photo_2021-09-11_08-06-21](https://user-images.githubusercontent.com/87842536/132934982-1773fc82-2b07-4661-9490-9f256699434a.jpg)

از اون آخری هم چیزی پیدا نکردم. فکر نمیکنم واقعی باشه

خلاصه اینکه اینا آدمای واقعین ولی فقط جلوی پروژه رو گرفتن که آقای اوسیوند دیده نشه

بحث nft کاملا مزخرفه و چند تا عکس فقط برای بازی دادنه

https://bscscan.com/token/0xdb238123939637d65a03e4b2b485650b4f9d91cb?a=0x9987605c8741d945098d7d6ba30bc41acc1b821e

این والتی هست که tastenft رو ساخته. خود همین والت میفروشه و میخره و رسما بازار دستشه! هفت درصد توکن ها داخل این والته. بیشتر از توکن هایی که داخل صرافی هست

جالب هم اینجاست که دقیقا با همین والت میاد تو صرافی تراکنش انجام میده

![photo_2021-09-11_08-07-51](https://user-images.githubusercontent.com/87842536/132935013-3a96aa7a-fc5b-47a4-8601-7559a0d21ead.jpg)

**قوانینی که نقض شده است:**
قانون ضد انحصار

**نوع هشدار:**
هشدار ریسک بالا

**نوع کلاهبرداری:** 
شید کوین، پامپ و دامپ

**وضعیت کلاهبرداری:**
در جریان

**نام کلاهبرداران:**
با معاونت آقای اوسیوند

**نام شرکت(ها):**

**آدرس وب سایت(ها):**
https://tastenfts.com/
https://t.me/TasteNFT
https://coinmarketcap.com/currencies/tastenft/
https://www.coingecko.com/en/coins/tastenft
https://bscscan.com/token/0xdb238123939637d65a03e4b2b485650b4f9d91cb?a=0x9987605c8741d945098d7d6ba30bc41acc1b821e
https://dex.guru/token/0xdb238123939637d65a03e4b2b485650b4f9d91cb-bsc

**برآورد مبلغ کلاهبرداری:**

**منابع:**
https://isthiscoinascam.com/check/tastenft

**جمع بندی:**

**Comment by samielec** [comment]
جرجندی کلاهبردار تو خودت زن و دخترای ایرانی رو توی دبی میفروختی حالا چون دستت رو شده این چرت و پرتا رو تحویل ملت میدی
